.github/workflows/node-pr-jobs.yml

```diff
@@ -2,55 +2,9 @@
 # License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
 name: Node
 
-on: [pull_request_target]
+on: [pull_request]
```

Comment:
I assume this is seperate so `pull_request` refers to ref/branch being PR'ed - ie no need to HEAD as per above? I assume there is no way that information could be discovered once and shared between jobs (so we only have one workflow)?

Reply:
Yep! I don't see the harm in these ones being normal `pull_request` workflow triggers as they don't need any secrets. It _might_ be possible to share the pr_head info as an artefact but given my previous fun with artefact downloading, I thought it would be best to leave that for now.